app/templates/community/index.hbs

```diff
@@ -1,122 +1,41 @@
-    </p>
-
-    <p>
-      Please do not report security vulnerabilities on the public GitHub issue tracker. The {{#link-to "security"}}Ember.js Security Policy{{/link-to}} details the procedure for disclosing security issues.
-    </p>
-
-    <p>
-      The source code for this website is also <a href="https://github.com/ember-learn/ember-website">available on GitHub</a>. If you have found a typo or incorrect documentation, please file an issue on the GitHub project page, or even better, submit a pull request.
-    </p>
-
-    <p>
-      Ember is an inclusive community that welcomes (and benefits from!) people of all backgrounds. If you're having a non-code issue, <a href="mailto:ombudsman@emberjs.com?subject=Website Help Link">reach out</a>. We do our very best to be sure every Ember user has pleasant interactions and positive experiences. If we can help make you feel more comfortable or help navigate a trying situation, let us know.
-    </p>
-  </div>
-</section>
-
-<section>
-  <h2 class="text-center">In a Giving Mood? Contribute to the Project</h2>
-
-  <div>
-    <img
-      alt="Contribute"
-      class="section-image pull-left"
-      src="/images/community/give.png"
-    >
-    <p>
-      The Ember.js source is hosted on <a href="https://github.com/emberjs/ember.js/">Github</a>. To contribute patches, create a fork of the project on GitHub and submit a pull request. Please be sure to include unit tests and documentation for any new features you add. See the <a href="https://github.com/emberjs/ember.js/blob/master/CONTRIBUTING.md">full guidelines</a> for more information.
-    </p>
-  </div>
-</section>
-
-<section class="section-split">
-  <div class="section-split-item text-center">
-    <h2 class="text-center">Feeling Lonely?<br>Join a Meetup</h2>
-
-    <img alt="Meetup" src="/images/community/meetup.png">
-    <p>
-      Joining an Ember.js meetup is a great way to learn more about Ember, meet other developers, find jobs, etc. Any and all things Ember and local.
-    </p>
-
-    {{#link-to "community.meetups" class="es-button"}}
-      Find a meetup
-    {{/link-to}}
-  </div>
-
-  <div class="section-split-item text-center">
-
-    <h2 class="text-center">Met The Mascots?<br>You'll Love Them!</h2>
-
-    <img alt="Campster" src="/images/community/campster.png">
-    <p>
-      Tomster and Zoey have been crafted with love, and are constantly getting makeovers. Events, User Groups, Countries&mdash;sky's the limit!
-    </p>
-    {{#link-to "mascots" class="es-button"}}
-      Meet the Mascots
-    {{/link-to}}
+<EsPagewrapper class="bg-muted">
```

Comment:
I wouldn't use `<EsPagewrapper>` in our implementation 🤔 if you take a look at the other pages in the landing page you'll see examples of how it should look 👍

Reply:
kk, will fix this asap